wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -178,20 +175,18 @@ private fun parseMessage(path: List<Int>, helper: SourceCodeHelper, message: Des
   val info = helper.getLocation(path)
 
   val nestedTypes = mutableListOf<TypeElement>()
-  val nestedPath = mutableListOf<Int>().apply { addAll(path) }
-  nestedPath.add(DescriptorProtos.DescriptorProto.NESTED_TYPE_FIELD_NUMBER)
-  nestedPath.add(0) // placeholder for index
+  val nestedMessagePath = mutableListOf<Int>().apply { addAll(path) }
```

Comment:
```suggestion
  val nestedMessagePath = mutableListOf(*path.toTypeArray())
```

This also works but I'm not a stickler for style